App/HejDTO/Sources/HejDTO/Contests/GET/Contests/Contests.Links.swift

```diff
@@ -0,0 +1,15 @@
+
+import Foundation
+
+extension Contests {
+    
+    public struct Links: Codable, Equatable {
```

Comment:
Could you merge all of those structs in one file? It will be more consistent with other ones. Please check out the other files :)

Reply:
Moved to others - wanted to reuse this one for the same Links in Contests